### ISSUE TITLE
Group of files added as a list

### ISSUE THREAD

**Issue by SantoshNII** [fyneworks/multifile#29]
Hi,

When we select group of files, on the UI, it is shown as a list. Out of the selected files if I have to delete a single file; it is not possible. We have to delete the complete list and then add.

Can this be done?

Thanks,
Santosh

**Comment by digggggggggg** [comment]
No. When you select a group of files they can only be removed as one (the entire group). This is because there is no way to manipulate the value of a file input element via javascript. It has to be what the user selected, or nothing at all.

**Comment by SantoshNII** [comment]
Thanks for the reply. But can you please have a look at this: http://jsfiddle.net/SantoshNII/ktLsb0md/1/ 
Here the feature which I asked for is implemented. But I have other issues.

Can the feature implemented in fiddle be included in the plugin?

**Comment by SantoshNII** [comment]
Hi digggggggggggg, 

Any update on this?

**Comment by fyneworks** [comment]
Hi @SantoshNII, I see what you did there... I think we can bring half of the functionality you wrote into the plugin. The file selection / removal side of it seems to work well. But the form submission aspect of it would force users down a particular path that might not be right for them. ie.: the plugin would no longer work with conventional forms (ie.: native submissions). Make sense?

**Comment by SantoshNII** [comment]
Perfect, thanks

**Comment by elshahat** [comment]
Any updates on SantoshNII request?
I also need this
